src/rustllvm/PassWrapper.cpp

```diff
@@ -431,7 +431,7 @@ extern "C" void LLVMRustConfigurePassManagerBuilder(
     bool MergeFunctions, bool SLPVectorize, bool LoopVectorize,
     const char* PGOGenPath, const char* PGOUsePath) {
   // Ignore mergefunc for now as enabling it causes crashes.
```

Comment:
remove this outdated comment, too?

Reply:
Oh yes… I'll wait for the try to finish and do the amend.